kombu/transport/redis.py

```diff
@@ -217,7 +217,7 @@ def _prefix_args(self, args):
         if command in self.PREFIXED_SIMPLE_COMMANDS:
             args[0] = self.global_keyprefix + str(args[0])
 
-        if command in self.PREFIXED_COMPLEX_COMMANDS.keys():
+        elif command in self.PREFIXED_COMPLEX_COMMANDS:
```

Comment:
Remove the line break please. It was there because these conditions where unrelated although you are right, they should be.

Reply:
fixed it.